content/en/docs/concepts/configuration/secret.md

```diff
@@ -854,11 +856,13 @@ kind: Secret
 metadata:
   name: secret-tls
 type: kubernetes.io/tls
-data:
+stringData:
```

Comment:
since stringData doesn't work that well with server side apply, I would avoid giving examples with it and would steer people toward using data

Reply:
Eesh. Is there a tracking issue for _`stringData` doesn't work that well with server side apply_, or for documenting that gap? (ideally: both)

Reply:
I see https://github.com/kubernetes/kubernetes/issues/118519, https://github.com/kubernetes/kubernetes/issues/96841, https://github.com/kubernetes/kubernetes/issues/89938#issuecomment-1033085353, but it deserves a callout in docs (both from server-side apply docs and from secret stringData docs)

Reply:
I filed https://github.com/kubernetes/website/issues/42603